prow/plank/report.go

```diff
@@ -154,32 +161,18 @@ func createEntry(pj kube.ProwJob) string {
 	}, " | ")
 }
 
-func prLink(pj kube.ProwJob) string {
-	refs := pj.Spec.Refs
-	var suffix string
-	if refs.Org == "kubernetes" {
-		if refs.Repo == "kubernetes" {
-			suffix = fmt.Sprintf("%d", refs.Pulls[0].Number)
-		} else {
-			suffix = fmt.Sprintf("%s/%d", refs.Repo, refs.Pulls[0].Number)
-		}
-	} else {
-		suffix = fmt.Sprintf("%s_%s/%d", refs.Org, refs.Repo, refs.Pulls[0].Number)
```

Comment:
I am not familiar with gubernator (yet) - is this the canonical URL that a project outside the k8s org is going to get by using gubernator?